src/util/serialization/format.hpp

```diff
@@ -34,6 +35,9 @@ namespace cbdc {
     /// \see \ref cbdc::operator<<(serializer&, std::byte)
     auto operator>>(serializer& packet, std::byte& b) -> serializer&;
 
+    auto operator<<(serializer& ser, const buffer& b) -> serializer&;
```

Comment:
Keeping inline with the rest of the serialization code, can these two methods be given doc comments?